OasisDEX/oasis-borrow
# POC Loading states for async simulation

# [POC Loading states for async simulation](https://app.shortcut.com/oazo-apps/story/8176/pisa-loading-states-for-async-simulation)

[Tab-1677155612813.webm](https://user-images.githubusercontent.com/16230404/220909387-8c8b5276-53c8-462c-8f31-a21a624984bc.webm)
  
## Changes 👷‍♀️

- Added animated skeleton loaders for afterpills,
- changed behavior or preloaders in order information to be displayed every time when user changes something, not just as initial state.
  
## How to test 🧪

Go to `/ajna/borrow/WBTC-USDC?network=goerli`, that's the only place where you could see it live.
